src/pages/docs.py

```diff
@@ -13,37 +13,33 @@
     - [Purpose of the Dashboard](#purpose-of-the-dashboard)
     - [Who Should Use This Dashboard?](#who-should-use-this-dashboard)
 
-    ### **2. Key Concepts**
-    - [Glossary](#glossary)
-        - [Basic Concepts](#basic-concepts)
-        - [Inspect AI Concepts](#inspect-ai-concepts)
-
-    ### **3. Metrics and Scoring**
+    ### **2. Glossary**
```

Comment:
thought it made sense to group all definitions of terms together in one Glossary section